src/components/views/rooms/RoomTile2.tsx

```diff
@@ -39,6 +39,14 @@ import * as FormattingUtils from "../../../utils/FormattingUtils";
  * warning disappears.                                             *
  *******************************************************************/
 
+enum NotificationColor {
+    // Inverted (None -> Red) because we do integer comparisons on this
+    None, // nothing special
```

Comment:
I think at least WebStorm doesn't like comments being after the entity, only on the line prior :(

Reply:
![image](https://user-images.githubusercontent.com/1190097/82741825-8fc1be80-9d13-11ea-9da9-752a1e3ee03f.png)

Seems fine to me.